plugins/filter_nest/nest.c

```diff
@@ -0,0 +1,284 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
```

Comment:
Please add missing header here:

```
/*  Fluent Bit
 *  ==========
 *  Copyright (C) 2015-2017 Treasure Data Inc.
```

Reply:
Added in 77ac1710893682fc9cb274539c3a1b8625807e95